docs/source/index.mdx

```diff
@@ -1,138 +1,67 @@
 
-```graphql title="Supergraph schema (simplified)"
-type User {
-  id: ID!
-  name: String!
-}
-
-type Product {
-  upc: String!
-  inStock: Boolean!
-}
-```
+The router receives incoming GraphQL operations from clients and intelligently routes them across your subgraphs. The router then receives subgraph responses and returns a single response to the client. **For clients, the request and response cycle looks exactly the same as querying any other GraphQL server.** No client-side configuration is required.
 
-> A real supergraph schema includes additional information that tells your router _which_ subgraph is responsible for _which_ types and fields. [Learn more about composition.](./federated-types/composition/)
+<Tip>
 
-### Server instances
+Learn more about [your router options](./building-supergraphs/router).
 
-In a federated architecture, each subgraph instance is a GraphQL service that's queried _only_ by the router. The router is a separate service that exposes a GraphQL endpoint to external clients. Clients query the router, and the router then queries individual subgraphs to obtain, combine, and return results:
+</Tip>
 
-```mermaid
-graph LR;
-  clients(Clients);
-  subgraph "Supergraph";
-  gateway([Router]);
-  serviceA[Users<br/>subgraph];
-  serviceB[Products<br/>subgraph];
-  gateway --- serviceA & serviceB;
-  end;
-  clients -.- gateway;
-  class clients secondary;
-```
-
-**The router** is one of the following:
-
-- [The Apollo Router](https://www.apollographql.com/docs/router/) (recommended): a high-performance, precompiled Rust executable
-    - If you're getting started with federation, we recommend [creating a **cloud supergraph**](/graphos/quickstart/cloud/) with Apollo GraphOS. With a cloud supergraph, GraphOS provisions and manages your router for you!
-- An instance of Apollo Server using special extensions from the [`@apollo/gateway`](/apollo-server/using-federation/api/apollo-gateway) library
-
-**Subgraphs** can each use any [subgraph-compatible GraphQL server library](./building-supergraphs/supported-subgraphs/).
+## Benefits of federation
 
-- This includes Apollo Server using special extensions from the [`@apollo/subgraph`](/apollo-server/using-federation/api/apollo-subgraph) library.
-- Different subgraphs in the same supergraph can use different server libraries.
+As a GraphQL-native microservices architecture, Apollo Federation draws on both paradigms' benefits.
+Federated architecture imparts some unique additional benefits by combining GraphQL and microservices frameworks.
+Continue reading to learn more.
 
+<Tip>
 
-## Benefits of federation
+- Learn more about the [considerations and benefits of GraphQL](/intro/benefits/).
+- Learn more about the [considerations and benefits of microservices architecture](https://aws.amazon.com/compare/the-difference-between-monolithic-and-microservices-architecture/).
```

Comment:
Also considered https://www.atlassian.com/microservices/microservices-architecture/microservices-vs-monolith, but AWS seemed like it would appeal more to our audience.